frontend/src/scenes/billing/billingLogic.ts

```diff
@@ -146,6 +147,45 @@ export const billingLogic = kea<billingLogicType>([
         billingAlert: [
             (s) => [s.billing, s.preflight, s.projectedTotalAmountUsd],
             (billing, preflight, projectedTotalAmountUsd): BillingAlertConfig | undefined => {
+                if (router.values.searchParams['products']) {
+                    let upgradedProducts = router.values.searchParams['products'].split(',')
+                    upgradedProducts = billing?.products.filter((product) => upgradedProducts.includes(product.type))
+                    const alerts: BillingAlertConfig[] = []
+                    upgradedProducts?.forEach((product: BillingProductV2Type) => {
+                        const currentPlan = product.plans.find((plan) => plan.current_plan)
+                        if (currentPlan?.initial_billing_limit) {
+                            alerts.push({
+                                status: 'warning',
+                                title: 'Billing Limit Automatically Applied',
+                                message: `To protect your and our costs, we've automatically applied a $${currentPlan?.initial_billing_limit} billing limit for ${product.name}.`,
```

Comment:
```suggestion
                                message: `To protect your costs and ours, we've automatically applied a $${currentPlan?.initial_billing_limit} billing limit for ${product.name}.`,
```

for some reason I think this reads better